src/settings/settings.html

```diff
@@ -9,6 +9,17 @@
 </head>
 
 <body>
+    <main class="main">
+        <section class="section">
+            <h1 class="section__header">Settings</h1>
+            <div class="section__input">
```

Comment:
Wouldn't it be easier with `input__wrapper` or `input__container`?

Reply:
Of course, I'm aware of the fact I said earlier `section__input` is enough so it is only a suggestion. You don't need to change it this time if you don't want to.